src/main/java/com/example/taxiservicespring/service/repository/impl/CarModelRepositoryImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.example.taxiservicespring.service.repository.impl;
+
+import java.util.List;
+
+import org.springframework.stereotype.Component;
+
+import com.example.taxiservicespring.service.model.CarModel;
+import com.example.taxiservicespring.service.repository.CarModelRepository;
+import com.example.taxiservicespring.util.DataGenerator;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Component
```

Comment:
Use @Repository annotation instead of @Component, it has actually no difference, but first one has more readability to reviewers